app/controllers/home_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HomeController < ApplicationController
   def index
   end
+  def show
+   @card= Card.all
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator '='.